eZ/Publish/Core/Event/ContentService.php

```diff
@@ -9,6 +9,32 @@
 namespace eZ\Publish\Core\Event;
 
 use eZ\Publish\API\Repository\ContentService as ContentServiceInterface;
+use eZ\Publish\API\Repository\Events\Content\AddRelationEvent as AddRelationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeAddRelationEvent as BeforeAddRelationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeCopyContentEvent as BeforeCopyContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeCreateContentDraftEvent as BeforeCreateContentDraftEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeCreateContentEvent as BeforeCreateContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeDeleteContentEvent as BeforeDeleteContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeDeleteRelationEvent as BeforeDeleteRelationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeDeleteTranslationEvent as BeforeDeleteTranslationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeDeleteVersionEvent as BeforeDeleteVersionEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeHideContentEvent as BeforeHideContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforePublishVersionEvent as BeforePublishVersionEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeRevealContentEvent as BeforeRevealContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeUpdateContentEvent as BeforeUpdateContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\BeforeUpdateContentMetadataEvent as BeforeUpdateContentMetadataEventInterface;
+use eZ\Publish\API\Repository\Events\Content\CopyContentEvent as CopyContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\CreateContentDraftEvent as CreateContentDraftEventInterface;
+use eZ\Publish\API\Repository\Events\Content\CreateContentEvent as CreateContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\DeleteContentEvent as DeleteContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\DeleteRelationEvent as DeleteRelationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\DeleteTranslationEvent as DeleteTranslationEventInterface;
+use eZ\Publish\API\Repository\Events\Content\DeleteVersionEvent as DeleteVersionEventInterface;
+use eZ\Publish\API\Repository\Events\Content\HideContentEvent as HideContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\PublishVersionEvent as PublishVersionEventInterface;
+use eZ\Publish\API\Repository\Events\Content\RevealContentEvent as RevealContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\UpdateContentEvent as UpdateContentEventInterface;
+use eZ\Publish\API\Repository\Events\Content\UpdateContentMetadataEvent as UpdateContentMetadataEventInterface;
```

Comment:
This long import list makes me feel that maybe `use eZ\Publish\API\Repository\Events\Content;` and e.g.
```php
$this->eventDispatcher->dispatch(
    new UpdateContentMetadataEvent($content, ...$eventData),
    Content\UpdateContentMetadataEvent::class
);
```
would be better.